regicide_game/player.py

```diff
@@ -0,0 +1,65 @@
+import random
+from typing import List
+
+from regicide_game.card import Card
+
+
+class Player:
+    hand: List(Card)
+
+    def __init__(self, max_hand_size, tavern_deck) -> None:
+        """
+        Deal cards to a player
+        """
+        if len(tavern_deck) > 0:
+            for i in range(min(max_hand_size, len(tavern_deck))):
+                self.hand.append(tavern_deck.pop(0))
+
+    def play_card(self) -> Card:
+        """
+        Plays a random card from the hand.
+        """
+        if len(self.hand) > 0:
+            played_card = random.choice(self.hand)
+            self.hand.remove(played_card)
+
+        return played_card
+
+    def discard_card(self, card_to_discard, discard_pile):
+        """
+        Discards a selected card to the discard pile.
+        """
+        self.hand.remove(card_to_discard)
+        discard_pile.append(card_to_discard)
+
+    def suffer_damage(self, enemy_card, discard_pile) -> bool:
+        """
+        Discard cards from the hand with a total value at least equal to the enemy's attack value.
+
+        Strategy of discarding: from left to right (simple, refine in the future)
+
+        Returns true if the player successfully takes damage, false if the game is over!
+        """
+        # determine the damage that will be dealt to you by the enemy
+        damage = enemy_card.power
```

Comment:
I think it would be easier to just pass in `damage` instead of passing in the enemy_card 🤔 what do you think?